src/backend/utils.ts

```diff
@@ -365,7 +365,8 @@ async function errorHandler(
   const otherErrorMessages = ['No saved credentials', 'No credentials']
 
   if (!window) {
-    window = BrowserWindow.getFocusedWindow() ?? BrowserWindow.getAllWindows[0]
+    window =
+      BrowserWindow.getFocusedWindow() ?? BrowserWindow.getAllWindows()[0]
```

Comment:
As an example on why these changes are necessary, take a look at this diff. Before, we were trying to access the first element of `BrowserWindow.getAllWindows`, which would of course never have worked since it's a function and not an array, but we weren't warned about that since `suppressImplicitAnyIndexErrors` was on